feed-ingress/cmd/elb.go

```diff
@@ -36,11 +38,40 @@ var elbCmd = &cobra.Command{
 	},
 }
 
+type loadBalancerTypeValue struct {
+	pflag.Value
+	value elb.LoadBalancerType
+}
+
+func (t *loadBalancerTypeValue) Set(s string) error {
+	if s == "standard" {
+		t.value = elb.Standard
+	} else {
+		t.value = defaultLoadBalancerType
+	}
+	return nil
+}
+
+func (t *loadBalancerTypeValue) Get() string {
+	if t.value == elb.Classic {
+		return "classic"
+	} else if t.value == elb.Standard {
+		return "standard"
```

Comment:
I'm just thinking about the terminology here.  I don't think AWS have a 'standard' load balancer, just 'NLB' and 'ALB'.  So should this be named NLB as we also have an ALB package?  This also makes me wonder if we should have an NLB package instead of adding NLBs to the ELB package... might be easier to discuss in person / call.

Reply:
You're right - and I like the idea of having an `nlb` subcommand - then we've got one subcommand for each of the AWS  load balancer types (Classic aka ELB, Network and Application)